packages/documentation/gatsby-config.js

```diff
@@ -86,5 +66,51 @@ module.exports = {
     `gatsby-transformer-remark`,
     `github-api-pages`,
     `mdx-pages`,
+    {
+      resolve: `@gatsby-contrib/gatsby-plugin-elasticlunr-search`,
+      options: {
+        // Fields to index
+        fields: [`title`, `tags`],
+        // How to resolve each field`s value for a supported node type
+        resolvers: {
+          // For any node of type MarkdownRemark, list how to resolve the fields` values
+          SitePage: {
+            title: node => {
+              if (node.context && node.context.frontmatter && node.context.frontmatter.title) {
+                return node.context.frontmatter.title;
+              }
+
+              return '';
+            },
+            tags: node => {
+              if (node.context && node.context.frontmatter && node.context.frontmatter.tags) {
+                return node.context.frontmatter.tags;
+              }
+
+              return '';
+            },
+            path: node => node.path,
+          },
+          Mdx: {
```

Comment:
Need to target specific node types. I don't know if `SitePage` and `Mdx` cover everything we want to index. Here's a list of all of the node types:

```'SitePage',
  'SitePlugin',
  'Site',
  'Directory',
  'File',
  'ImageSharp',
  'ComponentProp',
  'ComponentMetadata',
  'ComponentDescription',
  'Mdx',
  'MarkdownRemark',
  'SiteSearchIndex'
```